Cold-Gas-Thruster/Cold Gas Thruster PDD.tex

```diff
@@ -0,0 +1,510 @@
+and high design and fabrication complexity.
+
+\section*{Acknowledgements}
+The author would like to thank Dr. Garrick and Dr. Olles for their help in understanding the theory of nozzles and the review of the system before testing.
+
+
+\onecolumn
+\appendices{}
+\section{\textbf{Detailed Fall Timeline}}
+\centering
+\begin{tabular}{ |p{1.25cm}||p{11.0cm}| }
+  \hline
+  Week \# & Plan \\
+  \hline
+  1 & Introductions \& Fall plan discussion \\
+  \hline
+  2 & Beginner Project: Supersonic Nozzle Design  \\
+  & Design Phase: Test stand \\
+  \hline
+  3 & Beginner Project: Nozzle Printing and Testing on weekend \\
+  & Design Phase: Propulsion System \\
+  \hline
+  4 & Design Phase: Propulsion System \\
+  \hline
+  5 & Design Phase: Arduino and MATLAB design \\
+  \hline
+  6 & Design Phase: Arduino and MATLAB design \\
+  \hline
+  7 & Fabrication Phase: Material order and making Solidworks drawings\\
+  \hline
+  8 & Fabrication Phase: Machining \\
+  \hline
+  9 & Fabrication Phase: Machining and assembly \\
+  \hline
+  10 & Experimentation Phase: Test 1 \\
+  \hline
+  11 & Experimentation Phase: Test 2 \\
+  \hline
+  12 & Data Analysis Phase: Data Organization \\
+  \hline
+  13 & Data Analysis Phase: Noise and error analysis \\
+  \hline
+  14 & Data Analysis Phase: Performance Analysis \\
+  \hline
+  15 & Project Culmination Report writing \\
+  \hline
+  16 & Project Culmination Report finalization \\
+  \hline
+\end{tabular}
+
+\newpage
+\section{\textbf{Cold Gas Thruster Preliminary Bill of Materials}}
+
+\centering
+\begin{tabular}{ |p{4cm}||p{1.0cm}|p{1.5cm}|p{2.5cm}|p{2.0cm}|p{2.0cm}| }
+  %\hline
+  %\multicolumn{6}{|c|}{Cold Gas Thruster Bill of Materials} \\
+  \hline
+  Part & Cost & Quantity & Vendor & Part Number & Status \\
+  \hline
+  Tank & \$131.35 & 1 & McMaster Carr & 7822A11 & Not acquired \\
+  \hline
+  Regulator & \$182.50 & 1 & McMaster Carr & 49305K21 & Not acquired \\
+  \hline
+  Solenoid & \$100.49 & 1 & McMaster Carr & 4711K511 & Not acquired \\
+  \hline
+  Push-to-connect fitting & \$4.33 & 2 & McMaster Carr & 5779K116 & Not acquired \\
+  \hline
+  Aluminum Pipe & \$2.45 & 2 & McMaster Carr & 50785K172 & Not acquired \\
+  \hline
+  Nylon Pipe & \$8.90 & 1 & McMaster Carr & 5548K77 & Not acquired \\
+  \hline
+  Aluminum Sheet & \$7.00 & 1 & McMaster Carr & 8975K596 & Acquired \\
+  \hline
+  Aluminum T-Slot & \$6.49 & 7 & McMaster Carr & 47065T101 & Acquired \\
+  \hline
+  T-Slot Corner Frame & \$7.98 & 10 & McMaster Carr & 5537T51 & Acquired \\
+  \hline
+  T-Slot Length Frame & \$5.66 & 4 & McMaster Carr & 47065T235 & Acquired \\
+  \hline
+  \#8 Socket Head Cap Screws & \$6.58 & 1 [100] & McMaster Carr & 92196A194 & Acquired \\
+  \hline
+  Arduino & \$10.00 & 1 & RIT Construct & Unknown & Not Acquired \\
+  \hline
+  Grand Total & \$608.25 & N/A & N/A & N/A & N/A \\
+  \hline
+\end{tabular}
+
+\bibliography{CGT-PDD.bib}
```

Comment:
The bibliography does not appear because you have not made any citations in your document.
To make a citation, insert the command `\cite{document-name}`, where `document-name` is the name of the entry in your bib file (`nozzles`, for example).

Citations should be made any time you reference work external to this document, or facts that may not be common knowledge. Most research papers do a good job of this, so there are plenty of examples to follow.